dataservices/service/api/v1/users_data_delete_test.go

```diff
@@ -23,21 +23,24 @@ var _ = Describe("UsersDataDelete", func() {
 			context = NewTestContext()
 			context.RequestImpl.PathParams["userid"] = targetUserID
 			context.AuthenticationDetailsImpl.IsServerOutputs = []bool{true}
-			context.DataStoreSessionImpl.DeleteDataForUserOutputs = []error{nil}
+			context.DataStoreSessionImpl.DestroyDataForUserByIDOutputs = []error{nil}
```

Comment:
if we are infact switching to `Destroy` rather than `Delete` then seems the file should then also become `users_data_destroy_test`